lib/ffprober/ffmpeg/version.rb

```diff
@@ -0,0 +1,38 @@
+module Ffprober
+  module Ffmpeg
+    class Version
+      VERSION_REGEX = /^(ffprobe|avprobe|ffmpeg) version (\d+)\.?(\d+)\.?(\d+)*/
+      NIGHTLY_REGEX = /^(ffprobe|avprobe|ffmpeg) version (N|git)-/
+
+      def version
+        @version ||= Gem::Version.new(parse_version.join('.'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.